oaknational/Oak-Web-Application
# fix: update tab widths, tweak subject phase picker spacing

## Description

- Updated tab spacing and sizing to more closely match Figma designs
- Updated Subject Phase Picker spacing
- Fixed issue where certain subject icons would not appear

## How to test

1. Go to https://deploy-preview-1894--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/geography-primary/overview
2. You should see the tabs and subject phase picker match the [Figma design](https://www.figma.com/file/3cz9fb0ObFDLv1c212OtJv/High-fidelity-designs?node-id=1%3A367)
3. You should see the subject icon for Geography to the right of the curriculum title ("Primary Geography")
4. Click on the Subject Phase Picker subject control ("Subject / Geography")
5. You should see icons for all subjects
6. Open the Subject Phase Picker phase control ("Phase / Primary")
7. You should see side-by-side buttons for "Primary" and "Secondary"

## Screenshots

How it used to look (delete if n/a):
<img width="1092" alt="Screenshot 2023-09-11 at 16 19 01" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/abaa3a28-147c-498b-8c24-758a304fa40e">
<img width="1092" alt="Screenshot 2023-09-11 at 16 19 12" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/ae6cdc71-7fab-4468-8672-78d66c1f84b1">
<img width="1092" alt="Screenshot 2023-09-11 at 16 19 19" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/8aecd420-f0f7-4453-b417-a904a86f1b5d">

How it should now look:
<img width="1092" alt="Screenshot 2023-09-11 at 16 18 02" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/5f310160-3f94-4f00-83eb-b0469d7774ac">
<img width="1092" alt="Screenshot 2023-09-11 at 16 18 21" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/ec544303-c56e-4503-8e06-bce6643c3347">
<img width="1092" alt="Screenshot 2023-09-11 at 16 18 30" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/ab9ba953-3823-412a-bbe2-9cb550527079">



## Checklist
